prestart.d/00-update-riak-conf.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+sed -i -r "s#storage_backend = (.*)#storage_backend = $BACKEND#g" $RIAK_CONF
```

Comment:
We probably should have some if checks around this for making it optional in case it's unset (the default). @fdhadzh Would you mind adding that?